spec/requests/case_spec.rb

```diff
@@ -2,6 +2,30 @@
 require 'support/shared_examples_for_glimr'
 
 RSpec.describe GlimrApiClient::Case do
+  include_examples 'no fees', 'TT/2016/00001', 'ABC123'
+
+  subject { described_class.find('TT/2016/00001', 'ABC123') }
+
+  describe '#title' do
+    it 'returns the title' do
```

Comment:
Oops. That was a copy/paste error - I ended up overwriting the title I'd put in the first place!
I'll fix it.